menas/ui/components/run/runDetail.controller.js

```diff
@@ -36,18 +37,24 @@ sap.ui.define([
       this._detail = this.byId("detailPage");
       this._checkpointsTable = this.byId("Checkpoints");
 
+      this._eventBus.subscribe("menas", "resize", this.iframeHack, this);
+
       this.byId("runIconTabBar").attachSelect(function(oSelectEv) {
         if(oSelectEv.getParameter("selectedKey") === "lineage") {
-          const oFrame = $("#lineage_iframe");
-          oFrame.removeClass("lineageIframe");
-          setTimeout(function() {
-            //force re-render
-            oFrame.addClass("lineageIframe");
-          }.bind(this), 200);
+          this.iframeHack();
         }
       }.bind(this));
     },
 
+    iframeHack: function() {
```

Comment:
If it is a hack, maybe add a few comments to it so future generations know what was happening